libsolidity/formal/CHC.cpp

```diff
@@ -927,17 +927,6 @@ void CHC::internalFunctionCall(FunctionCall const& _funCall)
 	Expression const* calledExpr = &_funCall.expression();
 	auto funType = dynamic_cast<FunctionType const*>(calledExpr->annotation().type);
 
-	auto contractAddressValue = [this](FunctionCall const& _f) {
```

Comment:
I guess this is simply unused?

Reply:
This lambda was exactly the same as `SMTEncoder::contractAddressValue`.
I did not dig in the history, but I assume it is a relict of some refactoring/generalization.